html-forms_personal-profile-form/index.html

```diff
@@ -10,6 +10,28 @@
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
-    <section></section>
+    <section>
+     <form> <p>We want to get to know you! Please enter your personal information</p>
```

Comment:
I did this text as the label for the fieldset, I think since it describes the content of it is more accessible with the label attribute.